docs/input/docs/usage/examples.md

```diff
@@ -0,0 +1,19 @@
+# Build Script
+
+To use the Cake Svn addin in your Cake file simply import it. Then define a task.
```

Comment:
Missing point for "Cake.Svn" addin.

Reply:
Fixed

@x-jokay Ready for another review